_inc/client/searchable-modules/index.jsx

```diff
@@ -38,12 +39,16 @@ export const SearchableModules = withModuleSettingsFormHelpers(
 			}
 
 			// Only should be features that don't already have a UI, and we want to reveal in search.
-			const whitelist = [ 'contact-form', 'enhanced-distribution', 'json-api', 'notes' ];
+			const whitelist = [ 'contact-form', 'enhanced-distribution', 'json-api', 'notes', 'private' ];
 
 			const allModules = this.props.modules,
 				results = [];
 			forEach( allModules, ( moduleData, slug ) => {
 				if ( this.props.isModuleFound( slug ) && includes( whitelist, slug ) ) {
+					// The Private Sites gets its own card.
+					if ( 'private' === slug ) {
+						return results.push( <Private key={ slug } /> );
```

Comment:
Noting that when used in `_inc/client/security/index.jsx`, we passed `{ ...commonProps }` to the component. We don't do that here as it does not seem necessary for the toggle to work. The unique key, however, is necessary to avoid warnings inside that loop of modules.